grub.d/default/xx_mender_font_and_theme_handling.cfg

```diff
@@ -1,3 +1,7 @@
+pkgdatadir="${pkgdatadir:-/usr/share/grub}"
+
+. "${pkgdatadir}/grub-mkconfig_lib"
```

Comment:
this file always exists, right?

Reply:
Yes, it is a standard file included by several of the upstream scripts as well.